sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -0,0 +1,139 @@
+# C# Azure SDK Clients that Contain Protocol Methods
+
+## Introduction
+
+Most Azure SDK Clients expose methods that take 'model' parameters, C# classes which map to the input\output of a REST JSON API call.
+
+Imagine a service that stores information about pets, with a GetDog and SetDog pair of operations. 
+
+Pets are represented in the request by a JSON hash:
+
+```json
+{
+        "name": "Buddy",
+        "id": 2,
+        "color": "Brown"
+}
+```
+
+An API using models might be:
+
+```csharp
+        public class Pet
+        {
+                string Name { get; }
+                int Id { get; }
+                string Color { get; }
+        }
+
+        Task<Response<Pet>> GetDog(string dogName);
+        Task<Response> SetDog(Pet dog);
+```
+
+Some SDK Clients however expose methods which often take `RequestContent` parameters and return `Response` instead.
+
+For example:
+
+```csharp
+        Task<Response> GetDog(RequestContent requestBody, RequestOptions options = default);
+        Task<Response> SetDog(RequestContent requestBody, RequestOptions options = default);
+```
+
+Those methods are called '**protocol methods**', as they provide more direct access to the REST protocol used by the client library.
+
+This quick start is designed to review their use.
+
+## Usage
+
+The basic structure of calls with clients remain the same if they use protocol methods or methods with models:
+
+1. [Initialize your client](#1-initialize-your-client "Initialize Your Client")
+2. [Create a request](#2-create-a-request "Create a Request")
+3. [Send the request](#3-send-the-request "Send the Request")
+4. [Handle the response](#4-handle-the-response "Handle the Response")
+
+## 1. Initialize Your Client
+
+The first step in interacting with a service is to create a client instance. 
+
+```csharp
+using Azure.Pets;
+using Azure.Core;
+using Azure.Identity;
+
+const string endpoint = "http://localhost:3000";
+var credential = new AzureKeyCredential(/*..*/);
+var client = new PetsClient(endpoint, credential);
+```
+
+The only difference is that sometimes these clients will be more specific in the credential required ('AzureKeyCredential' or 'TokenCredential' for example).
+
+## 2. Create and Send a Request
+
+Protocol methods need a JSON object of the shape required by the service in question.
+
+See the specific service documentation for details, but as an example:
+
+```csharp
+        var data = new Dictionary<string, object> {
+                ["name"] = "Buddy",
+                ["id"] = 2,
+                ["color"] = "Brown"
+        };
+        client.SetDog(RequestContent.Create(data));
+```
```

Comment:
I know that you showed the expected JSOn above, but it's so far away that readers might lose the context. To solve this problem, maybe you could add a C# comment showing (again) the JSON payload.